tox.ini

```diff
@@ -223,27 +231,36 @@ commands =
         test/integration/ \
         {posargs}
 
+[testenv:pylint-examples]
+basepython = {[testenv:pylint]basepython}
+deps = {[testenv:pylint]deps}
+commands =
+    pylint --rcfile=examples/src/pylintrc examples/src/aws_encryption_sdk_cli_examples/
+    pylint --rcfile=examples/test/pylintrc --disable R0801 examples/test/
```

Comment:
nit: any reason not to disable R0801 directly in the pylintrc?

Reply:
i copy pasta'd what we did for the python esdk: https://github.com/aws/aws-encryption-sdk-python/blob/master/tox.ini#L167

Reply:
I'll go ahead and move it

Reply:
plus we are already disabling it in the pylintrc: https://github.com/aws/aws-encryption-sdk-cli/blob/28051dc6df9be97de2d3b2072ad7d4698ffec241/examples/test/pylintrc#L9